### ISSUE TITLE
Character encoding on Chrome

### ISSUE THREAD

**Issue by shaugh** [MaazAli/Meteor-HighCharts#42]
Strange character on tooltip for funnel hover. I found similar issues with `&nbsp`encoding in Chrome after a Chrome update - could be a similar issue?

(https://cloud.githubusercontent.com/assets/8186884/19753552/527d26f2-9c51-11e6-951d-36320f69fd84.png)

**Comment by jhuenges** [comment]
I dont think that this is an issue with this package. You should check if there already is an issue in the highcharts repository.
